setup.py

```diff
@@ -17,11 +17,12 @@
     name="pimsviewer",
     version='2.0',
     description="Viewer for Python IMage Sequence (PIMS).",
-    author="Casper van der Wel",
-    author_email="caspervdw@gmail.com",
+    author="Ruben Verweij",
```

Comment:
Probably should get @caspervdw to sign off on this change.

Reply:
@caspervdw I wanted to ask you. I rewrote the program from scratch to use Tk instead of Qt because it is easier to install pimsviewer then because Tk is installed with Python. Also, the interface code could then be put into an XML. Because the code is completely different now, I figured you don't want to be contacted about it for the 2.0 versions now ;-)

Reply:
If not, then I'll just revert it :-)

Reply:
Fine :+1:  @tacaswell  thanks for the heads-up

I am curious about the result @rbnvrw